app/clinic/controllers/dashboardController.js

```diff
@@ -15,7 +15,7 @@
     $scope.flags = [];
     $scope.patientUUID = $stateParams.patientUuid;
     $scope.todayVisit = null;
-    $scope.hasLabOrderPrivilege = false;
+    $scope.hasLabOrderPrivilege = true;
```

Comment:
If `authorizationService.hasPrivilege` fails for some reason will the feature be available even for users without privileges? If we change the opposite can also happen, not sure which one is better.

Reply:
Fixed @ynurmahomed